versioned_docs/version-8.4/components/console/manage-clusters/manage-api-clients.md

```diff
@@ -43,7 +43,6 @@ When the rate limit is triggered, the client will receive an HTTP 429 response.
 Currently, Camunda 8 SaaS supports the following scopes:
 
 - Zeebe - Access to the [Zeebe gRPC](/apis-tools/zeebe-api/grpc.md) API.
-- Tasklist - Access to the [Tasklist GraphQL](/apis-tools/tasklist-api/tasklist-api-overview.md) API.
```

Comment:
As we still support Tasklist V1, we should add here the reference to Tasklist API (https://docs.camunda.io/docs/apis-tools/tasklist-api-rest/tasklist-api-rest-overview/)